packages/zowe-explorer/tsconfig.json

```diff
@@ -4,13 +4,13 @@
     "target": "es6",
     "outDir": "out",
     "jsx": "react",
-    "lib": ["es6", "dom"],
```

Comment:
I assume we will bring this back when we make an attempt to support `vscode.dev` / Web

Reply:
Honestly I'm unsure if we really need `dom` here - but I think we should be using es2020 (if possible) according to the VS Code extension [helloworld-sample](https://github.com/microsoft/vscode-extension-samples/blob/877375478be5382d834ad9ff8dd1e5f7ddf8eb65/helloworld-sample/tsconfig.json#L5) - `es6` is ECMAScript 2015, might have some benefits to upgrading 🤷‍♂️ 

But I'll look into this a bit more before resolving 😋

Reply:
So I did see that some popular extensions such as GitLens put `dom` in the `lib` array, so I've added it back in commit fb8cf5d.